WebRoot/js/zimbraMail/prefs/view/ZmPreferencesPage.js

```diff
@@ -674,6 +674,9 @@ function(parentIdOrElem, text, width, listener, id) {
 	button.setText(text);
 	button.addSelectionListener(listener);
 	this._replaceControlElement(parentIdOrElem, button);
+	if(className) {
```

Comment:
Thanks for your input. But I feel that's an interval working of api `addClassName` which is very well abstracted from us.

Now here, we need to add class to `DwtControl` i.e button in our case. We don't have to remove class. So in my opinion `DwtControlInstance.addClassName("className")` api will be more appropriate and readable than `delClass` api. What do you think?

Also if you see, signature of `Dwt.delClass`, it needs DOM element as input. And here at this point, we have just created control, we have not injected in into DOM.

Let me know if you have any more question on this.

Thanks